stdlib/area/area.lua

```diff
@@ -314,4 +330,28 @@ function Area.tostring(area)
     return 'Area {'..left_top..', '..right_bottom ..'}'
 end
 
+--- Converts an entity and its collision_box to the area around it
+-- @see Entity.to_collision_area
+-- @tparam LuaEntity entity
+-- @treturn LuaBoundingBox
+function Area.to_collision_area(entity)
+    fail_if_missing(entity, "missing entity argument")
```

Comment:
I'm not against duplicates in general but this one does not make much sense. `Area.to_collision_area = Entity.to_collision_area` would be the same thing.